### PR TITLE
Functional tests - Refacto 'Orders > Shopping carts' tests

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Refacto Orders > Shopping carts tests
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | no need
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25108)

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/02_orders/05_shoppingCarts/01_filterShoppingCarts.js [hunk 1]
```diff
@@ -108,12 +108,14 @@ describe('Filter the Shopping carts table', async () => {
     await expect(numberOfShoppingCartsAfterReset).to.be.equal(numberOfShoppingCarts);
   });
 
-  it('should change pagination to 300 item per page', async function () {
-    await testContext.addContextItem(this, 'testIdentifier', 'changeItemNumberTo50', baseContext);
+  if (numberOfShoppingCarts < 21) {
+    it('should change pagination to 300 items per page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'changeItemNumberTo50', baseContext);
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/02_orders/05_shoppingCarts/01_filterShoppingCarts.js]:
```suggestion
      await testContext.addContextItem(this, 'testIdentifier', 'changeItemNumberTo300', baseContext);
```

**Review by Progi1984 — COMMENTED** [review]
A small feedback

File: tests/UI/campaigns/functional/BO/02_orders/05_shoppingCarts/01_filterShoppingCarts.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/02_orders/05_shoppingCarts/01_filterShoppingCarts.js]: Done thx :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh & @boubkerbribri

**Pull request merged**